style.css

```diff
@@ -0,0 +1,52 @@
+html, body {
+  padding: 0;
+  margin: 0;
+}
+
+h1 {

```

Comment:
This works here because the markup is very simple, but if this were a real site would you want **every** `h1` to be `inline-block` for instance? Might be best to use classes, almost always, before using just the elements in CSS.